modular_skyrat/modules/liquids/code/liquid_systems/liquid_plumbers.dm

```diff
@@ -27,6 +27,9 @@
 	var/is_pumping = FALSE
 	/// Floor tile is placed down
 	var/tile_placed = FALSE
+	
+	///category for plumbing RCD
+	category = "Liquids"
```

Comment:
are there defines for categories?

Reply:
There probably should be, but no. All the plumbing RCD 's do categories like this both here and upstream.